src/backend/distributed/planner/deparse_shard_query.c

```diff
@@ -350,6 +365,34 @@ UpdateRelationsToLocalShardTables(Node *node, List *relationShardList)
 }
 
 
+/*
+ * FindRelationShardByOid finds the RelationShard with given Oid.
```

Comment:
could mention it might return NULL, like:
>FindRelationShardByOid finds the RelationShard for shard relation with
given Oid if exists in given relationShardList. Otherwise, returns NULL.

Also, what about `FindRelationShardByShardRelationOid` ?, otherwise readers might think that we pass distributed table's Oid instead of shard relation's Oid

Reply:
updated the comment.
I think current name sounds more accurate, with Oid in general. Because this function is being called not only by shard relation id, but also with constraint Oid, and some other possible types of oid's in the future.

Reply:
>some other possible types of oid's in the future.

at that time, shouldn't we define seperate functions for them ?